### ISSUE TITLE
Add hamburger/dropdown menu

### ISSUE THREAD

**Issue by jamesjulich** [Badger-Spill/badger-spill.github.io#2]
The current menu we have should be replaced by a hamburger menu for better accessibility/aesthetics.
